deploy/vagrant/Vagrantfile

```diff
@@ -22,8 +22,11 @@ Vagrant.configure('2') do |config|
 
     provisioner.vm.network :private_network,
                         virtualbox__intnet: "tink_network",
+                        libvirt__network_name: "tink_network",
+                        libvirt__host_ip: "192.168.1.254",
                         libvirt__dhcp_enabled: false,
                         libvirt__forward_mode: 'none',
+                        libvirt__adapter: 1,
```

Comment:
This was needed to ensure that the private network isn't accidentally picked up as eth0 instead of eth1.